content/vendor/guides/airgapped-cluster.md

```diff
@@ -0,0 +1,184 @@
+---
+date: "2020-02-20T00:00:00Z"
+lastmod: "2020-02-20T00:00:00Z"
+title: "Airgapped Cluster"
+weight: "1006"
+---
+
+The KOTS Airgapped Cluster is an advanced guide that shows how to install a KOTS app in an Airgapped environment. This guide will assume you've already completed one of the [Getting Started Guides](/vendor/guides/#getting-started) to setup a non-airgapped cluster.
+
+It is broken into few sections:
+
+- [Download Airgap Bundle and License](#download-airgap-bundle-and-license)
+- [Embedded kURL Cluster](#embedded-kurl-cluster)
+- [Existing Kubernetes Cluster](#existing-kubernetes-cluster)
+- [Upload Airgap Bundle and License](#upload-airgap-bundle-and-license)
+- [Automatically Build Bundles](#automatically-build-bundles)
+- [Upload New Bundle](#upload-new-bundle)
+
+## Download Airgap Bundle and License
+
+Lets first download the `.airgap` bundle and license from [vendor.replicated.com](https://vendor.replicated.com).
+
+### Airgap Bundle
+
+To download the `.airgap` bundle, go to `[App Name] -> Channels -> [Channel Name] -> Release History`.
+
+![Airgap Channels](/images/guides/kots/airgap-channels.png)
+
+* Click on `Build` to build the `.airgap` bundle.
+
+* Once you see `Download Bundle`, click to download the `.airgap` bundle.
+
+![Airgap Download Bundle](/images/guides/kots/airgap-download-bundle.png)
+
+### Airgap License
+
+To download the airgap license, go to `[App Name] -> Customer -> [Customer Name]`.
+
+![Airgap Customers](/images/guides/kots/airgap-customers.png)
+
+* Select `License options -> Airgap Download Enabled` and `Save Changes`.
+
+* Click on `Download license` to re-download the airgapped enabled yaml license.
+
+![Airgap Download License](/images/guides/kots/airgap-download-license.png)
+
+## Embedded kURL Cluster
+
+Installing KOTS app into an embedded kURL kubernetes cluster needs a few additional steps.
```

Comment:
Let's try to be a bit more specific here, instead of `a few additional steps`, let's describe the steps:

```suggestion
Next, to install our KOTS app into an embedded kURL kubernetes cluster, we'll download the tar file containing our kubernetes components
```